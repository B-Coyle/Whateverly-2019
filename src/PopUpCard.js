import React, {Component} from 'react';
import Card from './Card.js';

class PopUpCard extends Card {
  constructor() {
    super();
    //camping: all information ex. nearest city
    //hiking: all info

    //will inherit from card
    //Two Options:
      //Option One:
      //EL: Show message "this many hiking/camping trails near you" that is clickable
        //if clicked, shows all camping information and hiking cards right off the bat
      //Option Two:
      //EL: Click button for more info
        //Displays pop-up
        //Another button with additional hiking/camping info, show nearby hikes/campsites

  }
  render {
    return (
    //CAMPSITE
    //Name
    //location
    //elevation (possible warning)
    //season
    //type
    //activities
    //reservation available

    //HIKING
    //name
    //location
    //elevation gain
    //length
    //difficulty
    //type
    )
  }
}

export default PopUpCard;