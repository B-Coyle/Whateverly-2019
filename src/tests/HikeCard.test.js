import React from 'react';
import ReactDOM from 'react-dom';
import PopUpCard from '../HikeCard/HikeCard.js';
import { shallow } from 'enzyme';


describe('HikeCard', () => {
let wrapper

beforeEach(() => {
    wrapper = shallow(
        <HikeCard />
    )
});

it('should mock snapshot', () => {
    expect(wrapper).toMatchSnapshot();
});

it('should have a default state', () => {
    expect(wrapper.state()).toEqual({
        favorite: false,
        visited: false,
        deleted: false,
    })
});
})