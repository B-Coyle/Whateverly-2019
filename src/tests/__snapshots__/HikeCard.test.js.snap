// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HikeCard should mock snapshot 1`] = `undefined`;
